schemas/metadata/pioneer-ingestion/pioneer-ingestion.1.schema.json

```diff
@@ -144,7 +149,6 @@
   "required": [
     "submission_timestamp",
     "metadata",
-    "pioneerId",
```

Comment:
Does jsonschema have an XOR type idiom that can be employed here? Otherwise it seems like you'd want a special case in the decoder to errorify messages missing both fields (though AIUI there's some special logic needed to pull ids out from elsewhere in the payload anyway so maybe this point is moot).

Reply:
Yes, this can be done via a `oneOf`. I hesitated because there is not support for it in the transpiler, but we don't even use the required information to propagate required/nullable information in BigQuery schemas.